File: trunk/venefica-webapp/src/main/java/com/venefica/model/Address.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.venefica.model;

import java.io.Serializable;
import javax.persistence.Column;
import javax.persistence.Embeddable;

/**
 *
 * @author gyuszi
 */
@Embeddable
public class Address implements Serializable {
    
    private String address1;
    private String address2;
    private String city;
    private String county;
    private String country;
    @Column(name = "statee")
    private String state;
    private String area;
    private String zipCode;
    
    public Address() {
    }
    
    public String getAddress1() {
        return address1;
    }

    public void setAddress1(String address1) {
        this.address1 = address1;
    }

    public String getAddress2() {
        return address2;
    }

    public void setAddress2(String address2) {
        this.address2 = address2;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getCounty() {
        return county;
    }

    public void setCounty(String county) {
        this.county = county;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    public String getArea() {
        return area;
    }

    public void setArea(String area) {
        this.area = area;
    }

    public String getZipCode() {
        return zipCode;
    }

    public void setZipCode(String zipCode) {
        this.zipCode = zipCode;
    }
}

File: branches/venefica-site-laravel/trunk/app/models/Address.php
<?php

class Address {
    
    public $id; //long
    public $name; //string
    public $address1; //string
    public $address2; //string
    public $city; //string
    public $county; //string
    public $country; //string
    public $stateAbbreviation; //string (2 chars)
    public $state; //string
    public $area; //string
    public $zipCode; //string
    public $longitude; //double
    public $latitude; //double
    
    
    public function __construct($obj = null) {
        
        if ( $obj != null ) {
        	foreach(get_class_vars(__CLASS__) as $k=>$v) {
        		if(isset($obj->$k))
					$this->$k = $obj->$k;
				elseif(is_array($obj) && isset($obj[$k]))
					$this->$k = $obj[$k];
        	}
        }
    }
	
}